### PR TITLE
Fixes to execute update_pgxn from pipeline manually

### PR DESCRIPTION
**Pull request by gurkanindibay**

### CODE REVIEW

File: .github/workflows/update-pgxn-version.yml [hunk 1]
```diff
@@ -16,21 +15,27 @@ jobs:
...
+      - name: Set git name and email
+        run: |
+          git config --global user.email "${{ github.event.inputs.microsoft_email }}" && \
+          git config --global user.name "${{ github.event.inputs.name }}"
+
       - name: Update pgxn files
         run: |
-          python -m packaging_automation.update_pgxn \
-          --prj_ver ${PROJECT_VERSION} \
+          python -m tools.packaging_automation.update_pgxn \
```
Comment (gurkanindibay) [inline .github/workflows/update-pgxn-version.yml:45]: Fixed path here

File: .github/workflows/update-pgxn-version.yml [hunk 2]
```diff
@@ -16,21 +15,27 @@ jobs:
     name: Update pgxn configuration
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out the repo
+        uses: actions/checkout@v2
 
       - name: Install dependencies
-        run: sudo apt install libcurl4-openssl-dev libssl-dev
+        run: sudo apt-get update && sudo apt-get install libcurl4-openssl-dev libssl-dev python3-testresources
```
Comment (gurkanindibay) [inline .github/workflows/update-pgxn-version.yml:30]: standardized this step with other pipelines

File: .github/workflows/update-pgxn-version.yml [hunk 3]
```diff
@@ -16,21 +15,27 @@ jobs:
     name: Update pgxn configuration
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Check out the repo
```
Comment (gurkanindibay) [inline .github/workflows/update-pgxn-version.yml]: standardized this step with other pipelines

File: .github/workflows/update-pgxn-version.yml [hunk 4]
```diff
@@ -16,21 +15,27 @@ jobs:
...
+        uses: actions/checkout@v2
 
       - name: Install dependencies
-        run: sudo apt install libcurl4-openssl-dev libssl-dev
+        run: sudo apt-get update && sudo apt-get install libcurl4-openssl-dev libssl-dev python3-testresources
 
       - name: Clone Tools branch
-        run: git clone --depth 1 --branch v0.8.1 https://github.com/citusdata/tools.git
+        run: git clone --depth 1 --branch update_properties_fixes https://github.com/citusdata/tools.git
```
Comment (gurkanindibay) [inline .github/workflows/update-pgxn-version.yml]: After merging https://github.com/citusdata/tools/pull/183, I will tag tools and will change here

File: .github/workflows/update-pgxn-version.yml [hunk 5]
```diff
@@ -16,21 +15,27 @@ jobs:
...
 
       - name: Install Python requirements
         run: python -m pip install -r tools/packaging_automation/requirements.txt
 
+      - name: Set git name and email
+        run: |
+          git config --global user.email "${{ github.event.inputs.microsoft_email }}" && \
+          git config --global user.name "${{ github.event.inputs.name }}"
```
Comment (hanefi) [inline .github/workflows/update-pgxn-version.yml:41]: You do not have these inputs defined above.

File: .github/workflows/update-pgxn-version.yml [hunk 6]
```diff
@@ -16,21 +15,27 @@ jobs:
...
+
       - name: Update pgxn files
         run: |
-          python -m packaging_automation.update_pgxn \
-          --prj_ver ${PROJECT_VERSION} \
+          python -m tools.packaging_automation.update_pgxn \
+          --prj_ver ${{ github.event.inputs.project_version }} \
           --gh_token ${GH_TOKEN} \
```
Comment (hanefi) [inline .github/workflows/update-pgxn-version.yml]: Why do you not have quotation marks here?

```suggestion
          --gh_token "${GH_TOKEN}" \
```

Please go through all the file for similar issues.

File: .github/workflows/update-pgxn-version.yml (hunk 3 above)
Comment (hanefi) [reply .github/workflows/update-pgxn-version.yml]: Is this the correct spelling though?

**Review by hanefi — CHANGES_REQUESTED** [review]

File: .github/workflows/update-pgxn-version.yml (hunk 3 above)
Comment (gurkanindibay) [reply .github/workflows/update-pgxn-version.yml]: You're right fixed it

File: .github/workflows/update-pgxn-version.yml (hunk 5 above)
Comment (gurkanindibay) [reply .github/workflows/update-pgxn-version.yml:41]: Added the inputs.

File: .github/workflows/update-pgxn-version.yml (hunk 6 above)
Comment (gurkanindibay) [reply .github/workflows/update-pgxn-version.yml]: Fixed

**Review by hanefi — APPROVED** [review]
Don't forget to update the branch name after https://github.com/citusdata/tools/pull/183 is merged

**Pull request merged**